src/Api/FingerprintApi.php

```diff
@@ -159,36 +148,45 @@ public function getEventWithHttpInfo($request_id)
                 }
             }
 
+            $serialized = ObjectSerializer::deserialize($content, $returnType, []);
+            $serialized->setRawResponse($responseBody);
+
             return [
-                ObjectSerializer::deserialize($content, $returnType, []),
+                $serialized,
                 $response->getStatusCode(),
                 $response->getHeaders()
             ];
 
         } catch (ApiException $e) {
             switch ($e->getCode()) {
                 case 200:
+                    $responseBody = $e->getResponseBody();
```

Comment:
PHP world prefers snake_case for variables

Reply:
Actually that's really debatable
https://www.php-fig.org/psr/psr-12/#7-closures

Let's we as Fingerprint choose a style for PHP because seems like world has no clear standard for this, as you can see in the link both `closureWithArgsVarsAndReturn` and `noArgs_longVars` are acceptable

Reply:
Regarding code standards, maybe it would be useful to integrate this tool into CI?
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer

Reply:
Just added php-cs-fixer as pre-commit hook, please re-review this approach

Reply:
LGTM.
Consider also integration this tool into CI: https://github.com/PHPCSStandards/PHP_CodeSniffer/
It's great to have git hook for fixing the PHP-CS issues, but we need to enforce these standards in CI as well.

Reply:
Let's have this CI change in another PR 👍